Paco/AndroidManifest.xml

```diff
@@ -365,5 +365,5 @@
     <uses-permission android:name="android.permission.READ_CALENDAR" />
 
     <uses-feature android:name="android.hardware.camera" android:required="false"/>
-    <uses-sdk android:minSdkVersion="14"  android:targetSdkVersion="22"></uses-sdk>
+    <uses-sdk android:minSdkVersion="16"  android:targetSdkVersion="22"></uses-sdk>
```

Comment:
Is there something in this change that requires sdk 16? If not, please revert so we are supporting back to Android 4.0.